doc/tutorial/concepts/custom_validator_solution2.cpp

```diff
@@ -0,0 +1,55 @@
+#include <seqan3/argument_parser/all.hpp>
+#include <cmath>
```

Comment:
I would include this into the snippet! Especially the cmath include

Reply:
this should be outdates